pages/manage_and_operate/api/first-steps/guide.en-gb.md

```diff
@@ -195,6 +195,13 @@ client = ovh.Client(
 print("Welcome", client.get('/me')['firstname'])
 ```
 
+#### List and revoke your keys
+
+There is currently no Control Panel to list and revoke your keys. The API portal can be used to do that : 
+* List the keys ids through [GET /me/api/application](https://eu.api.ovh.com/console-preview/?section=%2Fme&branch=v1#get-/me/api/application)
```

Comment:
you could use formatted links to the console, like this: `@api {v1} /me GET /me/api/application/{applicationId}`, what do you think ?